### ISSUE TITLE
Add content description for login items

### ISSUE THREAD

**Issue by eliserichards** [mozilla-mobile/fenix#18318]
## Context
When a user opens the Saved Logins screen, they will see a list of their logins. Each login they see is a `logins_item.xml`. We should add a `contentDescription` to `logins_item.xml` for screen readers. 

## Acceptance criteria
1. Add a `contentDescription` to the two `TextView`s in `logins_item.xml`:
   * Web address: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/layout/logins_item.xml#L24
   * Username: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/layout/logins_item.xml#L41
2. The `contentDescription` should be a string added to [strings.xml](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/values/strings.xml)
   * Note: there is a `strings.xml` file for each language. Make sure you are adding it to the main string file in /main/res/values/strings.xml
   * Make sure your string has a description ([example](https://github.com/mozilla-mobile/fenix/blob/dd09ba64c4d13ccd1a3ade639a16321f6c04a988/app/src/main/res/values/strings.xml#L12)) 

Example of a content description: https://github.com/mozilla-mobile/fenix/blob/dd09ba64c4d13ccd1a3ade639a16321f6c04a988/app/src/main/res/layout/fragment_edit_login.xml#L46

<img width="958" alt="image" src="https://user-images.githubusercontent.com/43795363/112671617-69d3e180-8e30-11eb-8861-ce13114edb5d.png">

**Comment by priyanka0906** [comment]
may  I work on this?

**Comment by AnishaSingh0118** [comment]
hello, as an outreachy applicant and a complete beginner in open source community, I really want to work on this bug could you guide me with my first contribution to open source.

**Comment by priyanka0906** [comment]
> may I work on this?

I am also an outreachy applicant.

**Comment by AnishaSingh0118** [comment]
Oh sorry do I commented on your comment?

**Comment by AnishaSingh0118** [comment]
Oh so do I commented on your comment?

**Comment by AnishaSingh0118** [comment]
Oh, nice to hear that, have you started your contribution?

**Comment by eliserichards** [comment]
Hey @priyanka0906 and @AnishaSingh0118! I'm so excited for you to begin your contributions! Since @priyanka0906 commented first, I think you can pick up this issue. 

Feel free to ask any questions here, in your Pull Request, or on the [Fenix Matrix channel ](https://chat.mozilla.org/#/room/#fenix:mozilla.org)

**Comment by priyanka0906** [comment]
> Hey @priyanka0906 and @AnishaSingh0118! I'm so excited for you to begin your contributions! Since @priyanka0906 commented first, I think you can pick up this issue.
> 
> Feel free to ask any questions here, in your Pull Request, or on the [Fenix Matrix channel ](https://chat.mozilla.org/#/room/#fenix:mozilla.org)

Thankyou  for assigning.

**Comment by priyanka0906** [comment]
> Oh, nice to hear that, have you started your contribution?
@AnishaSingh0118 not yet ....this would me my first contribution  for outreachy.

**Comment by AnishaSingh0118** [comment]
On Wed, 24 Mar 2021, 2:04 am Elise Richards, ***@***.***>
wrote:

> Hey @priyanka0906 <https://github.com/priyanka0906> and @AnishaSingh0118
> <https://github.com/AnishaSingh0118>! I'm so excited for you to begin
> your contributions! Since @priyanka0906 <https://github.com/priyanka0906>
> commented first, I think you can pick up this issue.
>

Can you suggest me some other beginner friendly issues that I can work on,
it would be really helpful.

**Comment by priyanka0906** [comment]
@eliserichards content description is to be added to each view in logins_item.xml ?

**Comment by eliserichards** [comment]
@priyanka0906 only for web address and username: 

    Web address: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/layout/logins_item.xml#L24
    Username: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/layout/logins_item.xml#L41

**Comment by mcarare** [comment]
Closing this as it does not follow a11y best practices.
